server/policy.go

```diff
@@ -67,6 +82,36 @@ func (s *Server) registerPolicyRoutes(g *echo.Group) {
 		return nil
 	})
 
+	g.DELETE("/policy/environment/:environmentID", func(c echo.Context) error {
+		id, err := strconv.Atoi(c.Param("policyID"))
```

Comment:
Shall we use environmentID instead as this is what the ":environmentID" for?

Reply:
I missed this, updated